cypress/integration/03.TagsAndCategories.feature

```diff
@@ -14,28 +14,27 @@ Feature: Tags and Categories
   looking at the popularity of a tag.
 
   Background:
-    Given we have a selection of tags and categories as well as posts
-    And my user account has the role "administrator"
-    Given I am logged in
+    Given my user account has the role "admin"
+    And we have a selection of tags and categories as well as posts
+    And I am logged in
 
   Scenario: See an overview of categories
     When I navigate to the administration dashboard
     And I click on "Categories"
     Then I can see a list of categories ordered by post count:
-      | Icon | Name               | Post Count |
-      |      | Just For Fun       | 5          |
-      |      | Happyness & Values | 2          |
-      |      | Health & Wellbeing | 1          |
+      | Icon | Name               | Posts      |
+      |      | Just For Fun       | 2          |
+      |      | Happyness & Values | 1          |
+      |      | Health & Wellbeing | 0          |
 
   Scenario: See an overview of tags
     When I navigate to the administration dashboard
     And I click on "Tags"
-    Then I can see a list of tags ordered by user and post count:
-      | # | Name        | Nutzer | Beiträge |
-      | 1 | Naturschutz | 2      | 2        |
-      | 2 | Freiheit    | 2      | 2        |
-      | 3 | Umwelt      | 1      | 1        |
-      | 4 | Demokratie  | 1      | 1        |
+    Then I can see a list of tags ordered by user count:
+      | # | Name      | Users  | Posts    |
+      | 1 | Democracy | 2      | 3        |
+      | 2 | Ecology   | 1      | 1        |
+      | 3 | Nature    | 1      | 2        |
```

Comment:
@appinteractive cucumber features are now based on factories. That sometimes exposes strange behaviour: Why is this table not ordered by user **and** post count?

Reply:
Good question, I think that was not possible at that time to sort by more then one param.